src/utils/validations/TrlValidations.ts

```diff
@@ -28,49 +28,49 @@ export const trlValidation = commonSchema.keys({
   frameDescription: Joi.string().valid(...FRAME_DESCRIPTION).optional().allow(null, ''),
   authIntoService: authIntoService,
   lettersOfAuth: lettersOfAuth,
-  make: Joi.string().max(30).required(),
-  model: Joi.string().max(30).required(),
+  make: Joi.string().max(30).allow(null, ''),
+  model: Joi.string().max(30).allow(null, ''),
   grossEecWeight: Joi.number().min(0).max(99999).optional().allow(null),
   axles: Joi.array().items(axlesSchema.keys({
     weights: weightsSchema.keys({
       eecWeight: Joi.number().min(0).max(99999).optional().allow(null)
-    }).required(),
+    }),
     brakes: Joi.object().keys({
       brakeActuator: Joi.number().min(0).max(999).optional().allow(null),
       leverLength: Joi.number().min(0).max(999).optional().allow(null),
       springBrakeParking: Joi.boolean().optional().allow(null, '')
     }).optional().allow(null, '')
-  })).min(1).required(),
-  roadFriendly: Joi.boolean().required(),
+  })).min(1),
+  roadFriendly: Joi.boolean().allow(null, ''),
   tyreUseCode: Joi.string().max(2).optional().allow(null, ''),
   brakes: brakesSchema.keys({
     loadSensingValve: Joi.boolean().optional().allow(null, ''),
-    antilockBrakingSystem: Joi.boolean().required()
-  }).required(),
+    antilockBrakingSystem: Joi.boolean().allow(null, '')
+  }),
   dimensions: Joi.object().keys({
     length: Joi.number().min(0).max(99999).optional().allow(null),
     width: Joi.number().min(0).max(99999).optional().allow(null),
     axleSpacing: Joi.array().items(Joi.object().keys({
       value: Joi.number().min(0).max(99999).optional().allow(null),
       axles: Joi.string().optional()
     })).optional()
-  }).required(),
+  }),
   frontAxleToRearAxle: Joi.number().min(0).max(99999).optional().allow(null),
-  rearAxleToRearTrl: Joi.number().min(0).max(99999).required(),
-  couplingCenterToRearAxleMin: Joi.number().min(0).max(99999).required(),
-  couplingCenterToRearAxleMax: Joi.number().min(0).max(99999).required(),
-  couplingCenterToRearTrlMin: Joi.number().min(0).max(99999).required(),
-  couplingCenterToRearTrlMax: Joi.number().min(0).max(99999).required(),
-  centreOfRearmostAxleToRearOfTrl: Joi.number().min(0).max(99999).required(),
+  rearAxleToRearTrl: Joi.number().min(0).max(99999).allow(null, ''),
+  couplingCenterToRearAxleMin: Joi.number().min(0).max(99999).allow(null, ''),
+  couplingCenterToRearAxleMax: Joi.number().min(0).max(99999).allow(null, ''),
+  couplingCenterToRearTrlMin: Joi.number().min(0).max(99999).allow(null, ''),
+  couplingCenterToRearTrlMax: Joi.number().min(0).max(99999).allow(null, ''),
+  centreOfRearmostAxleToRearOfTrl: Joi.number().min(0).max(99999).allow(null, ''),
```

Comment:
```suggestion
  rearAxleToRearTrl: Joi.number().min(0).max(99999).allow(null),
  couplingCenterToRearAxleMin: Joi.number().min(0).max(99999).allow(null),
  couplingCenterToRearAxleMax: Joi.number().min(0).max(99999).allow(null),
  couplingCenterToRearTrlMin: Joi.number().min(0).max(99999).allow(null),
  couplingCenterToRearTrlMax: Joi.number().min(0).max(99999).allow(null),
  centreOfRearmostAxleToRearOfTrl: Joi.number().min(0).max(99999).allow(null),
```

Reply:
I get what you're doing but what's the point? This sucks anyway, empty strings are converted to null in dynamo